pkg/security/utils/cgroup.go

```diff
@@ -17,10 +17,14 @@ import (
 	"strings"
 
 	"github.com/DataDog/datadog-agent/pkg/security/common/containerutils"
+	"github.com/DataDog/datadog-agent/pkg/security/secl/model"
 )
 
 // ContainerID is the type holding the container ID
-type ContainerID string
+type ContainerID model.ContainerID
+
+// ContainerFlags is the type holding the container flags
+type ContainerFlags uint64
```

Comment:
I would prefer to have a type here in case we change the underlying type.
An other option is to define a model.CGroupFlags type and remove the one from pkg/security/utils/cgroup.go completely. WDYT ?

Reply:
If we change the underlying type we will probably have to change other things. I would rather use an alias if possible and limit the number of conversion ?

Reply:
We mix cgroups and containers in many places so tthis idea is to have a compilation error when miximg cgroup and container ids. Agreed offline with @spikat to keep it